kahuna/public/js/image/view.html

```diff
@@ -190,7 +192,7 @@
                ui:drag-data="ctrl.image | asImageAndCropsDragData:ctrl.crop"
                ui:drag-image="extremeAssets.smallest | assetFile">
                 <!-- TODO: Add tracking to crop -->
-                <img class="easel__image"
+                <img class="easel__image checkered__background"
```

Comment:
As I've recently learned thanks to Shaun and Akash, we're trying to use [BEM](https://css-tricks.com/bem-101/) for selectors. 

so i _think_ this becomes 

``` HTML
<img class="easel__image easel__image--checkered__background"
```